ibis/backends/base/sql/alchemy/__init__.py

```diff
@@ -885,3 +885,75 @@ def drop_view(
 
         with self.begin() as con:
             con.execute(view)
+
+
+class AlchemyCrossSchemaBackend(BaseAlchemyBackend):
```

Comment:
Would you mind adding short (possibly just one line) docstrings on:

- the class, indicating how this differs from the `BaseAlchemyBackend`
- The asbtractmethod below, indicating what a subclass should do to satisfy the contract.

Reply:
Done!